composer.json

```diff
@@ -81,7 +81,7 @@
         "filp/whoops": "^2.4",
         "guzzlehttp/guzzle": "^6.3",
         "league/flysystem-cached-adapter": "^1.0",
-        "mockery/mockery": "^1.2.3",
+        "mockery/mockery": "^1.3.1",
```

Comment:
Against 6.x? Can't we make a requirement >= 1.2.3 < 1.3.0 and > 1.3?

Reply:
Doesn't matter, since this is only a dev-dependency.